<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(159, 159, 159);</string>
  </property>
  <widget class="QLineEdit" name="l_sum_credit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_time">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_rate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ann">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color : white;
color: rgb(33, 33, 33);



</string>
   </property>
   <property name="text">
    <string>         Аннуитетные</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_diff">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color : white;
color: rgb(33, 33, 33);



</string>
   </property>
   <property name="text">
    <string>    Дифференцированные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="count">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(110, 165, 197);
color: rgb(33, 33, 33);
  border: 2px solid gray;

}

QPushButton:pressed {
 background-color: qradialgradient(spread:pad, cx:0.504926, cy:0.431818, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 139, 214, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_month_payment">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_overpayment">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_total_payment">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Общая сумма кредита :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Срок :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>мес</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Процентная ставка :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Тип платежей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Общая выплата :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>№ месяца получения :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_month">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-alignment: 'AlignRight';
border-bottom: 5px solid rgb(247, 255, 183);
background-color : white;
color: rgb(84, 84, 85);
	border-color: rgb(255, 110, 106);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
  background-color: white;
color: rgb(84, 84, 85);
  border: 3px solid gray;

}

QPushButton:pressed {
 background-color: qradialgradient(spread:pad, cx:0.504926, cy:0.431818, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 139, 214, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Credit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
